styleguide/_themes/global/scss/grid.scss

```diff
@@ -16,3 +26,12 @@
     width: $container-lg;
   }
 }
+
+.container,
+.container-fluid {
+  position: relative;
```

Comment:
I'm also nervous. A good bit of our general content has containers and container-fluid in them.